src/frontend/replayserver.cc

```diff
@@ -135,7 +136,16 @@ int main( void )
         }
 
         if ( best_score > 0 ) { /* give client the best match */
-            cout << HTTPResponse( best_match.response() ).str();
+            string best_response = HTTPResponse( best_match.response() ).str();
+            // Remove first line if status line is first
+            int pos = best_response.find( "\n" );
+            string first_line = best_response.substr( 0, pos );
+            if ( first_line.find( "HTTP" ) == 0 ) {
+                int start_pos = first_line.find( ' ' );
+                cout << "Status: " << first_line.substr( start_pos+1, 3 );
```

Comment:
Seems like the `Status: <blah>` way of communicating the status message is the CGI way (and not the NPH + CGI way) of passing status information to the underlying CGI execution engine. (https://tools.ietf.org/html/rfc3875#section-6.3.3)

mod_cgi handles headers generated by NPH + CGI scripts completely differently that I don't understand despite spending sometime looking at the source code.